lib/Parser.js

```diff
@@ -200,11 +219,7 @@ Parser.prototype = {
      * @private
      */
     _pushScope: function (scope, parentScope) {
-        if (!scope) {
-            throw new Error('Scope cannot be blank');
-        }
-
-        this._scopes.push({scope: scope, parentScope: parentScope});
+        this._scopes.push({scope: scope || '', parentScope: parentScope});
```

Comment:
We could allow this, but for some cases we still need to throw
`throw new Error('Scope cannot be blank'); `